pipelines/ocw/pipeline-ocw-www.yml

```diff
@@ -79,6 +66,18 @@ jobs:
               mkdir ../ocw-hugo-themes/theme
               tar -xvzf ../ocw-hugo-themes/ocw-hugo-themes-*.tgz -C ../ocw-hugo-themes/theme
               hugo --config ../ocw-hugo-projects/ocw-www/config.yaml --themesDir ../ocw-hugo-themes/theme
+      - task: copy-s3-buckets
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source: {repository: amazon/aws-cli, tag: latest}
+          run:
+            path: sh
+            args:
+            - -exc
+            - |
+              aws s3 sync s3://((ocw-studio-bucket)) s3://((ocw-bucket)) --exclude '*' --include '((ocw-www-site))/*'
```

Comment:
I don't think we want to just copy over everything in the ocw studio bucket, as this could end up publishing an asset that isn't ready to be released yet. We should try for a more targeted sync command.

Reply:
That's what the `--include '((ocw-www-site))/*'` is doing: only files specific to this site will be copied over.